src/domains/user/accounts.routes.integration.spec.js

```diff
@@ -18,10 +18,122 @@ describeDbTestSuite('accountsRouter', () => {
         .post('/signup')
         .set('Content-type', 'application/json')
         .send(createAccountBody);
+      const { message, token } = response.body;
       expect(response.body).toEqual({
-        message: 'account created',
+        message,
+        token,
       });
       expect(response.status).toEqual(201);
     });
+
+    test('it should not create an account without email', async () => {
+      const createAccountBody = {
+        password: '0PasSwoRd12',
+      };
+
+      const response = await request(app)
+        .post('/signup')
+        .set('Content-type', 'application/json')
+        .send(createAccountBody);
+      const { message } = response.body;
+      expect(response.body).toEqual({
```

Comment:
please test for the exact response message that is returned from the app

Reply:
cool